cg-charter.html

```diff
@@ -183,11 +184,12 @@ <h3 id="specifications">
 
     <dd>An API as described above.</dd>
 
-    <dt>GPU for the Web Shading Language</dt>
+    <dt>WebGPU Shading Language</dt>
 
-    <dd>A Shading Language specification, probably based on an existing
-        language, that defines the programmable interface to the
-        GPU for the Web graphics and computation pipeline.</dd>
+    <dd>A Shading Language specification that defines the programmable
+        interface to the GPU for the Web graphics and computation
+        pipeline, and that can be translated or compiled into
+        platform-specific instructions.</dd>
```

Comment:
A possible addition might be similar to the API side we make an explicit point about representability in the shading languages of modern APIs. This currently could be read as WGSL will be compiled directly to machine code, but I might be trying to view it too naively for the purpose of reviewing.

Reply:
@alan-baker The actual WG charter might be a better place for the feedback- I am just reflecting that in this case! thank you cc: @tidoust